test/libsolidity/syntaxTests/conversion/implicit_conversion_of_super_in_comparison.sol

```diff
@@ -0,0 +1,24 @@
+contract D {}
+
+contract C {
```

Comment:
Should this be `contract C is D`?

Reply:
No. For completeness I wanted the test to include both a comparison that was causing and ICE and one that was not. Both should result in the same compiler error.

I added comments in the file to make it clearer that `D` is meant to be an unrelated contract on purpose.